lib/dslash_coarse.cu

```diff
@@ -4,6 +4,7 @@
 #include <worker.h>
```

Comment:
Lots of formatting changes for no apparent reason in this file.

Reply:
Had an offline discussion that not fixing this one is fine.